ci/test_wheel_distributed_ucxx.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+PROJECT_NAME="distributed_ucxx"
+
+source "$(dirname "$0")/test_utils.sh"
+source "$(dirname "$0")/test_common.sh"
+
+mkdir -p ./dist
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+RAPIDS_PY_WHEEL_NAME="${PROJECT_NAME}_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
+
+# Install previously built ucxx wheel
+RAPIDS_PY_WHEEL_NAME="ucxx_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./local-ucxx-dep
+python -m pip install ./local-ucxx-dep/ucxx*.whl
+
+# echo to expand wildcard before adding `[extra]` requires for pip
+python -m pip install $(echo ./dist/${PROJECT_NAME}*.whl)[test]
+
+# TODO: We need distributed installed in developer mode to provide test utils,
+# we still need to match to the `rapids-dask-dependency` version.
+rapids-logger "Install Distributed in developer mode"
+git clone https://github.com/dask/distributed /tmp/distributed
+python -m pip install -e /tmp/distributed
+
+# Run smoke tests for aarch64 pull requests
+# if [[ "$(arch)" == "aarch64" && "${RAPIDS_BUILD_TYPE}" == "pull-request" ]]; then
+#     python ci/wheel_smoke_test.py
+# else
+#     python -m pytest ./python/${PROJECT_NAME}/tests -k 'not test_sparse_pca_inputs' -n 4 --ignore=python/cuml/tests/dask && python -m pytest ./python/${PROJECT_NAME}/tests -k 'test_sparse_pca_inputs' && python -m pytest ./python/cuml/tests/dask
+# fi
```

Comment:
It looks like we're running the full test suite on aarch64 and not just smoke tests. We run smoke tests instead of a full test suite for ARM wheels testing of pull requests for some RAPIDS libraries because we want to minimize ARM runner usage. We _do_ run the full test suite on ARM for conda packages and in the nightly builds for wheels, so we still have coverage (just not in PRs for wheels).

If we intend to run the full test suite on ARM for PRs, we should remove this commented section. Otherwise we should implement a minimal set of "smoke tests" in `ci/wheel_smoke_test.py`. This could be as simple as ensuring the package can be imported.

Reply:
Done in https://github.com/rapidsai/ucxx/pull/167/commits/3ef814b2ca7340d9fa4b6fda3f4624ed2d7cd323 .